CK3toEU4/Source/CK3World/World.cpp

```diff
@@ -125,11 +125,17 @@ void CK3::World::verifySave(const std::string& saveGamePath)
 		saveGame.saveType = SaveType::ZIPFILE;
 	else
 	{
-		saveFile.seekg(0);
+		saveFile.seekg(0, saveFile.end);
```

Comment:
What's the idea behind using end as a member variable instead of std::ios_base::beg ?